### DISCUSSION TITLE
how to use sass

### DISCUSSION THREAD

**Discussion by fedorkanan** [community/community#64936] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

pls help me whit  sass

**Comment by naserianali** [comment]
1- you should install sass 
for more information you can read this document https://sass-lang.com/install/
2- you can use this commend for run live compiler sass --watch and after this you need to give it scss or sass directory location and then css location

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
